docs/DeveloperGuide.md

```diff
@@ -8,30 +29,194 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Main Mode
+
+This section describes the implementation of Main Mode features.
+
+Once the `main()` method of ComputerComponentChooser is called, objects for the `BuildManager`, `Parser`, 
+`editParser`, `Storage`, `Ui` are created.
+
+
+
+#### BuildManager
+
+![](/images/BuildManager.png)
```

Comment:
Please kindly remove the c icon! otherwise, good work on the diagram!

![image](https://user-images.githubusercontent.com/84378807/198253640-6fd8da24-8974-4f02-9e12-b11eb6c005fe.png)

Reply:
You may do so by adding in 'hide circle'